### ISSUE TITLE
Seems after upgrade Nzbhydra2 no longer has browser access

### ISSUE THREAD

**Issue by BytesBacon** [theotherp/nzbhydra2#699]
Only recently have I switched from version 1 to version 2. Everything has been running great until there was an update.  (v3.13.1 to v3.13.1)

The service is up and running but I can't access web-ui, getting back 502 (nginx reverse proxy setup, nor can access locally on port 5076).

I've re-installed everything, thinking it was just a corrupt upgrade, and deleted the database (/opt/nzbhydra2/data) just in case.

And still can't get it up for accessing it with the web-ui. 

Are there any other files that nzbhydra2 on Linux builds? (That would need to be deleted for a "clean" re-install?)

Is there an issue with having an old version of nzbhydra installed but not running? (/opt/nzbhydra/)

Thanks

Here's the nzbhydra2.log

[nzbhydra2.log](https://github.com/theotherp/nzbhydra2/files/6206510/nzbhydra2.log)

**Comment by dphildebrandt** [comment]
I'm having the same issue, except I'm using Traefik v2.  Using Docker, with `linuxserver/nzbhydra2`. I notice when I bring it out from behind the proxy, the docker container is reachable again.

I spoke with the maintainer briefly in this [ Reddit thread.](https://www.reddit.com/r/nzbhydra/comments/mbjz36/nzbhydra2_container_502_intermittently/)

See attached debug log with HTTP/S markers.
[nzbhydra-debuginfos-2021-03-25-12-10.zip](https://github.com/theotherp/nzbhydra2/files/6206551/nzbhydra-debuginfos-2021-03-25-12-10.zip)

**Comment by theotherp** [comment]
None of the changes made in the last months could cause this problem...

**Comment by BytesBacon** [comment]
I've removed version 1 completely, removed version 2 (unless there are other files built outside of /opt/nzbhydra2/) and download version v3.13.1 (downgraded back the version before the auto-upgrade) and still hitting error 502. (Before version v3.13.1 had zero issues)

I'm not sure why it's not working.

@theotherp do you happen to know all of the paths that nzbhydra2 uses on linux? (Or which source file to look at to check for all the paths. Wondering if I have some other corrupt file.)

And is there a switch for debugging? (Wondering if it's a conflict with some other dependencies that may have changed as well.)

Thanks

**Comment by theotherp** [comment]
Hydra only stores data in its data folder. You can change the level for the log file and console to "DEBUG".

By the way, the log says `http://127.0.1.1:5076`, that looks wrong...

**Comment by BytesBacon** [comment]
Yeah, I see that now too.

I'm not 100% what happened initially but I've got it up and running again (clean install and setup).

For anyone else that reads this in the future. Seems that when upgrading from nzbhydra v1 to v2, version 2 pulls the config file from version 1 (nice feature). But this means it's using port 5075 (version 1) where version 2 is on 5076.

So after re-installing version 2 as a clean install, I needed to fix my reverse proxy server with the "right" port and updated it as stated in the wiki.

My thoughts, this seemed more to be with upgrading from version 1 to version 2 and then version 2 upgrading itself if I had to guess. Everything seems to be working now and just need to rebuild the config file manually now.

Thanks @theotherp for pointing that out, that at least pointed me in the right direction to fixing it with a clean install. Have a great weekend and keep up the great work on the project, version 2 loads 10 times faster than version 1.
